tests/test_s3.py

```diff
@@ -114,6 +115,107 @@ def test_metrics(self):
             'AWS/S3.NumberOfObjects.Average' in resources[0]['c7n.metrics'])
 
 
+class AccessFilter(BaseTest):
+
+    recording = os.environ.get('C7N_RECORD') == '1'
```

Comment:
when we want to enable this via env var, I imagine we'll just do it directly in the replay setup, I'd rather not multiple paths here so that we can switch once and have it uniform across the tests.